include/full_coverage_path_planner/spiral_stc.hpp

```diff
@@ -23,22 +23,43 @@
 #include <geometry_msgs/msg/pose_stamped.hpp>
 #include <nav_msgs/msg/path.hpp>
 #include <nav_msgs/srv/get_map.hpp>
+#include "visualization_msgs/msg/marker.hpp"
 
 using namespace std::chrono_literals;
 using std::string;
+
 namespace full_coverage_path_planner
 {
   class SpiralSTC : public nav2_core::GlobalPlanner , private full_coverage_path_planner::FullCoveragePathPlanner
   {
   public:
+
```

Comment:
are these variables parameters? if so, please add a comment and group them in a block. Also, internal variables are declared under the protected label of the class (You do not want a user to access them from outside the class)

Reply:
Indeed, I have added the ones that will definitely stay to the node parameters and wrote an explanation for them in the README file in the latest commit. The remaining debug and testing parameters are grouped and moved under the protected label of the class.